source/api/presentation/3.0/index.md

```diff
@@ -155,7 +155,7 @@ Other properties are allowed, either via custom extensions or endorsed by IIIF.
 ###  3.1. Descriptive Properties
 
 ##### label
-A human readable label, name or title for the resource. This property is intended to be displayed as a short, textual surrogate for the resource if a human needs to make a distinction between it and similar resources, for example between pages or between a choice of images to display.
```

Comment:
To avoid multiple "thus" I suggest starting second sentence with "The `label` property"

Reply:
Seems that I tied comment to the deleted line when it is intended to apply to the replacement